src/constants/sfx.asm

```diff
@@ -2,278 +2,278 @@
+MUSIC_AFTER_BOSS                        equ $18
+MUSIC_BOSS                              equ $19
+MUSIC_INTRO_CUTSCENE                    equ $1A
+MUSIC_OBTAIN_INSTRUMENT                 equ $1B
+MUSIC_INTRO_WAKE_UP                     equ $1C ; Marin & Tarin's House
+MUSIC_OVERWORLD_SWORDLESS               equ $1D
+MUSIC_DREAM_SHRINE_SLEEP                equ $1E
+MUSIC_SOUTHERN_SHRINE                   equ $1F
+MUSIC_FULL_MOON_CELLO                   equ $20
+MUSIC_2D_UNDERGROUND                    equ $21
+MUSIC_OWL                               equ $22
+MUSIC_FINAL_BOSS                        equ $23
+MUSIC_DREAM_SHRINE_BED                  equ $24
+MUSIC_HEART_CONTAINER                   equ $25
+MUSIC_CAVE                              equ $26
+MUSIC_OBTAIN_POWERUP                    equ $27
+MUSIC_CONCH_HORN                        equ $28
+MUSIC_SEA_LILYS_BELL                    equ $29
+MUSIC_SURF_HARP                         equ $2A
+MUSIC_WIND_MARIMBA                      equ $2B
+MUSIC_CORAL_TRIANGLE                    equ $2C
+MUSIC_ORGAN_OF_EVENING_CALM             equ $2D
+MUSIC_THUNDER_DRUM                      equ $2E
+MUSIC_MARIN_SING                        equ $2F
+MUSIC_MANBOS_MAMBO                      equ $30
+MUSIC_OVERWORLD_INTRO                   equ $31 ; After obtaining sword
+MUSIC_MR_WRITE_HOUSE                    equ $32
+MUSIC_ULRIRA                            equ $33
+MUSIC_TARIN_BEES                        equ $34
+MUSIC_FROG_SONG                         equ $35
+MUSIC_MONKEYS_BUILDING_BRIDGE           equ $36
+MUSIC_CHRISTINE_HOUSE                   equ $37
+MUSIC_TOTAKA_UNUSED                     equ $38
+MUSIC_TURTLE_ROCK_ENTRANCE_BOSS         equ $39
+MUSIC_FISHERMAN_UNDER_BRIDGE            equ $3A
+MUSIC_OBTAIN_ITEM_UNUSED                equ $3B
+MUSIC_FILE_SELECT_TOTAKA                equ $3C ; Accessible only in German DX and Japanese versions. Enter file name MOYSE/とたけけ
+MUSIC_ENDING                            equ $3D
+MUSIC_MOBLIN_HIDEOUT                    equ $3E
+MUSIC_ISLAND_DISAPPEAR                  equ $3F
+MUSIC_RICHARD_HOUSE                     equ $40
+MUSIC_EGG_BALLAD_DEFAULT                equ $41 ; Play without instrument #3
+MUSIC_EGG_BALLAD_BELL                   equ $42 ; Play with instrument #3
+MUSIC_EGG_BALLAD_HARP                   equ $43 ; Play with instruments #3-4
+MUSIC_EGG_BALLAD_MARIMBA                equ $44 ; Play with instruments #3-5
+MUSIC_EGG_BALLAD_TRIANGLE               equ $45 ; Play with instruments #3-6
+MUSIC_EGG_BALLAD_ORGAN                  equ $46 ; Play with instruments #3-7
+MUSIC_EGG_BALLAD_ALL                    equ $47 ; Play with instruments #3-8
+MUSIC_GHOST_HOUSE                       equ $48
+MUSIC_ACTIVE_POWER_UP                   equ $49
+MUSIC_LEARN_BALLAD                      equ $4A
+MUSIC_CATFISHS_MAW                      equ $4B
+MUSIC_OPEN_ANGLERS_TUNNEL               equ $4C
+MUSIC_MARIN_ON_BEACH                    equ $4D
```

Comment:
Original comment stated that the beginning and end notes are skipped

Reply:
What I think it's referring to is that 4D sound very similar to a part in the 4E song but they're not identical. I don't see it worth pointing out.